client/src/workspace/layouts/AppsViewLayout.tsx

```diff
@@ -20,10 +20,28 @@ export const AppsViewLayout: React.FC = () => {
   const portalApp = findAppById(data, app.definition.id);
 
   const icon = portalApp?.icon || app.definition.notes.icon || 'Generic-App';
-  const userGuideLink =
```

Comment:
We should be able to use this same code, and either expand it by also checking `portalApp?.bundle_user_guide_link`, or adjusting the backend response to include `userGuideLink` for html entries